hera_cal/utils.py

```diff
@@ -259,17 +259,18 @@ def JD2RA(JD, longitude=21.42830, epoch='current'):
     -----------
     JD : type=float, a float or an array of Julian Dates
 
-    longitude : type=float, longitude of observer in degrees east, default=HERA long
+    longitude : type=float, longitude of observer in degrees east, default=HERA longitude
+
+    latitude : type=float, latitude of observer in degrees north, default=HERA latitutde
+               This only matters when using epoch="J2000"
     
-    epoch : type=str, epoch for RA calculation. options=['current'].
+    epoch : type=str, epoch for RA calculation. options=['current', 'J2000'].
             The 'current' epoch is the epoch at JD. Note that
-            LST is *defined* as the zenith RA in the current epoch. 
-
-    ## TODO : add a "J2000" option to return RA in J2000 epoch.
+            LST is defined as the zenith RA in the current epoch. 
```

Comment:
Though it's pedantic, I think that it's worth mentioning that it's not _technically_ J2000, but rather ICRS, that we're phasing up to. The difference of right ascension is evidently only ~78 mas (per wikipedia: https://en.wikipedia.org/wiki/International_Celestial_Reference_System), but worth noting. I _do_ think phasing to ICRS is the better choice, since it's an externally-defined system, so just adding a comment here is sufficient.

Reply:
ok good point